core/src/i18n/en.yaml

```diff
@@ -960,6 +962,7 @@ jobs:
     status: Status
     title: Conditions
     type: Type
+  description: ' is a finite or batch task that runs to completion.'
```

Comment:
What do you mean by "runs to completion"?

Reply:
It runs until it is terminated. It is taken from the K8s glossary. Maybe it will be better to write sth like: `creates one or more Pods and ensures that a specified number of them successfully terminate.`?

Reply:
I think more specific info could be useful here, 'cause I'd generally assume that any task eventually runs to completion and any Pod eventually terminates (hopefully successfully 🤞 ), so I'd try to emphasize what's special here.